cfme/v2v/migration_plans.py

```diff
@@ -0,0 +1,626 @@
+import attr
+import csv
+import tempfile
+import time
+from selenium.common.exceptions import StaleElementReferenceException
+
+from navmazing import NavigateToAttribute, NavigateToSibling
+from selenium.webdriver.common.keys import Keys
+from widgetastic.exceptions import NoSuchElementException
+from widgetastic.widget import Checkbox, View, ConditionalSwitchableView
+from widgetastic_manageiq import (
+    HiddenFileInput,
+    InfraMappingList,
+    MigrationPlansList,
+    MigrationProgressBar,
+    MigrationPlanRequestDetailsList,
+    MigrationDashboardStatusCard,
+    MigrationDropdown,
+    RadioGroup,
+    Table,
+    V2vPaginatorPane,
+)
+from widgetastic_patternfly import Button, BootstrapSelect, Text, TextInput, SelectorDropdown
+
+from cfme.exceptions import ItemNotFound
+from cfme.modeling.base import BaseCollection, BaseEntity
+from cfme.utils.appliance.implementations.ui import navigator, CFMENavigateStep, navigate_to
+from cfme.utils.log import logger
+from cfme.utils.version import Version, VersionPicker
+from cfme.utils.wait import wait_for
+
+from . import MigrationView
+
+
+class DashboardCards(View):
+    not_started_plans = MigrationDashboardStatusCard("Not Started")
+    in_progress_plans = MigrationDashboardStatusCard("In Progress")
+    completed_plans = MigrationDashboardStatusCard("Complete")
+    archived_plans = MigrationDashboardStatusCard("Archived")
```

Comment:
@sshveta  `archived_plans` from `L40` points to same thing, you can remove it.

Reply:
:+1: